templates/financial-services/resources.html

```diff
@@ -0,0 +1,253 @@
+{% extends "financial-services/_base_financial-services.html" %}
+
+{% block title %}Financial services | Resources{% endblock title %}
+{% block meta_description %}Explore a collection of useful resources that will help you understand how financial institutions can leverage secure and supported open source to accelerate their digital transformation journeys. {% endblock meta_description %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1FIOi8l7N2On3KVVscIzWUwthhYvdkU21IhanBDUwlCI/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru">
+  <div class="u-fixed-width">
```

Comment:
I think we usually limit the width to `col-7` or something when the title is too long. 
```suggestion
  <div class="row">
    <div class="col-7">
```